pkg/ring/kv/multi.go

```diff
@@ -0,0 +1,357 @@
+package kv
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/go-kit/kit/log/level"
+	"github.com/uber-go/atomic"
+
+	"github.com/cortexproject/cortex/pkg/util"
+)
+
+var (
+	primaryStoreGauge = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "consul_multikv_primary_store",
```

Comment:
It sounds a bit weird prefixing these metrics with `consul_`. What's the rationale behind it?

Reply:
Ehm, that was supposed to be `cortex` :)